api/src/main/kotlin/com/openlattice/collaborations/CollaborationsApi.kt

```diff
@@ -0,0 +1,97 @@
+package com.openlattice.collaborations
+
+import com.openlattice.organizations.OrganizationDatabase
+import retrofit2.http.*
+import java.util.*
+
+interface CollaborationsApi {
+
+    companion object {
+        // @formatter:off
+        const val  SERVICE = "/datastore"
+        const val  CONTROLLER = "/collaborations"
+        const val  BASE = SERVICE + CONTROLLER
+        // @formatter:on
+
+        const val DATABASE_PATH = "/database"
+        const val ORGANIZATIONS_PATH = "/organizations"
+        const val PROJECT_PATH = "/project"
+        const val TABLES_PATH = "/tables"
```

Comment:
Not actually sure what we standardize on across the codebase but I think its more clear to use `FOO_PATH` to designate path variables (`/{FOO}`) and `FOO` to designate constant paths (`/foo`)

Reply:
I think this makes most sense: `FOO_PATH` is for the path and `FOO_PARAM` is for params.